.github/workflows/python-tests.yaml

```diff
@@ -0,0 +1,60 @@
+name: Python Packages Test
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - python/sqlcommenter-python/**
+  pull_request:
+    paths:
+      - python/sqlcommenter-python/**
+
+jobs:
+  unittests:
+    env:
+      # We use these variables to convert between tox and GHA version literals
+      py36: 3.6
+      py37: 3.7
+      py38: 3.8
+      py39: 3.9
+      RUN_MATRIX_COMBINATION: ${{ matrix.python-version }}-${{ matrix.package }}-${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false  # ensures the entire test matrix is run, even if one permutation fails
+      matrix:
+        python-version: [ py36, py37, py38, py39 ]
+        os: [ ubuntu-20.04 ]
+    steps:
+      - name: Checkout Contrib Repo @ SHA - ${{ github.sha }}
+        uses: actions/checkout@v2
+      - name: Set up Python ${{ env[matrix.python-version] }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env[matrix.python-version] }}
+      - name: Install tox
+        run: pip install -U tox-factor
+      - name: run tox
+        run: tox -e '${{ matrix.python-version }}-{django21,django22,django30,django31,django32,psycopg2,flask,generic,sqlalchemy}'
+        working-directory: ./python/sqlcommenter-python
+
+  misc:
```

Comment:
Is `misc` a good name for this job? it's only linting right now ... are there plans to add more operations to this job?